src/libraries/Microsoft.Extensions.Logging.Console/tests/Microsoft.Extensions.Logging.Console.Tests/ConsoleLoggerConfigureOptions.cs

```diff
@@ -25,7 +25,7 @@ public void EnsureConsoleLoggerOptions_ConfigureOptions_SupportsAllProperties()
             Assert.Equal(3, typeof(ConsoleFormatterOptions).GetProperties(flags).Length);
             Assert.Equal(5, typeof(SimpleConsoleFormatterOptions).GetProperties(flags).Length);
             Assert.Equal(4, typeof(JsonConsoleFormatterOptions).GetProperties(flags).Length);
-            Assert.Equal(4, typeof(JsonWriterOptions).GetProperties(flags).Length);
+            Assert.Equal(6, typeof(JsonWriterOptions).GetProperties(flags).Length);
```

Comment:
Why did this assertion need to be changed?

Reply:
Because we have added 2 public properties to the `JsonWriterOptions` class.
BTW, I am not sure whether the associated comment is impacting us here.
```cs
// When adding a new property to one of these types, ensure the corresponding
// IConfigureOptions class is updated for the new property.
````